src/net/udpclient.ts

```diff
@@ -36,7 +37,20 @@ class UdpClient {
   public bind() : Promise<net.Endpoint> {
     // TODO: not sure what else this should be?
     return this.socket.bind('127.0.0.1', 0)
-        .then((endpoint:net.Endpoint) => {
+        .then((resultCode:number) => {
```

Comment:
This file has been deleted, so you might want to re-base :-)

Sorry I didn't do this earlier...